src/sentry/south_migrations/0338_fix_null_user_last_active.py

```diff
@@ -0,0 +1,885 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models, transaction
+
+
+class Migration(DataMigration):
+    def forwards(self, orm):
+        db.commit_transaction()
+        try:
+            self._forwards(orm)
+        except Exception:
+            # Explicitly resume the transaction because
+            # South is going to try and roll it back, but when
+            # it can't find one, it'll error itself, masking
+            # the actual exception being raised
+            #
+            # See https://github.com/getsentry/sentry/issues/5035
+            db.start_transaction()
+            raise
+        db.start_transaction()
+
+
+    def _forwards(self, orm):
+        from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+        User = orm['sentry.User']
+
+        queryset = User.objects.all()
+        for user in RangeQuerySetWrapperWithProgressBar(queryset):
+            with transaction.atomic():
+                cutoff_time = datetime.datetime(2017, 6, 29, 23, 44)              
+                User.objects.filter(
+                    id=user.id,
+                ).select_for_update().update(
```

Comment:
You can just do `update`

Something like this is what you need:

```python
User.objects.filter(
  id=user.id,
  last_active__is_null=True,
).update(last_active=timezone.now())
```

I don't fully see what you're intending to do here though since you're running `last_active=user.last_active` which won't change anything. But the code above is something closer to your intent I believe.